SU-HSDO/suhumsci
# ARCH-55 hover menu and libraries

# READY FOR REVIEW

# Summary
- Added all hover menu related Twig templates and refactored as needed for sub-theme
- Added all hover menu SCSS and JS files as needed and refactored as needed for sub-theme
- Removed FontAwesome now that it is being called in stanford_basic
- Added new libraries and call them as needed

# Needed By (Date)
- End of sprint (5/9)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Ensure that hover menu works
3. Ensure that FontAwesome is no longer a library or being imported via NPM

# Affected Projects or Products
- Archaeology
- suhumsci
- su_humsci_theme

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/ARCH-55

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
